tests/e2e/inversify.config.ts

```diff
@@ -76,7 +77,9 @@ import { IApiUrlResolver } from './utils/workspace/IApiUrlResolver';
 const e2eContainer: Container = new Container({ defaultScope: 'Transient' });
 
 e2eContainer.bind<IDriver>(TYPES.Driver).to(ChromeDriver).inSingletonScope();
+e2eContainer.bind<ITestWorkspaceUtil>(CLASSES.WorkspaceUtil).to(TestWorkspaceUtil);
```

Comment:
sorry, missed this. `e2eContainer.bind<ITestWorkspaceUtil>(CLASSES.WorkspaceUtil).to(TestWorkspaceUtil);`
should be `e2eContainer.bind<ITestWorkspaceUtil>(TYPES.WorkspaceUtil).to(TestWorkspaceUtil);` and `WorkspaceUtil` should be removed from CLASSES and added into TYPES.

Reply:
These changes have been applied in previous commits after removing the Interface.